Kwc/Form/NonAjax/Component.defer.js

```diff
@@ -0,0 +1,8 @@
+var onReady = require('kwf/commonjs/on-ready');
+var Form = require('kwf/commonjs/frontend-form/form');
+
+onReady.onRender('.kwcClass', function form(form) {
+    if (!form.get(0).kwcForm) {
+        new Form(form);
```

Comment:
vat foo = new Form()
foo.submit = function() {
    this.el.find('form').submit();
}